tests/fixtures/napari.py

```diff
@@ -49,4 +49,42 @@ def _valid_dataset_with_localised_nans(nan_location):
 
         return (out_path, valid_poses_dataset)
 
-    return _valid_dataset_with_localised_nans
+    return _valid_poses_dataset_with_localised_nans
+
+
+@pytest.fixture
+def valid_poses_dataset_long(valid_poses_dataset, tmp_path):
+    """Return a (path, dataset) pair representing a poses dataset
+    with 2 individuals ("id_0" and "id_1") and 3 keypoints
+    ("centroid", "left", "right") moving in uniform linear
+    motion for 10 frames in 2D space.
+    """
+    # Export as a DLC-csv file
+    out_path = tmp_path / "ds_long.csv"
+    save_poses.to_dlc_file(
+        valid_poses_dataset, out_path, split_individuals=False
+    )
+
+    return (out_path, valid_poses_dataset)
+
+
+@pytest.fixture
+def valid_poses_dataset_short(valid_poses_dataset, tmp_path):
+    """Return a valid poses dataset with 2 individuals and 3 keypoints
+    moving in uniform linear motion for 5 frames in 2D space.
+
+    The dataset is modified from the `valid_poses_dataset` which represents
+    2 individuals ("id_0" and "id_1") with up to 3 keypoints ("centroid",
+    "left", "right") moving in uniform linear motion for 10 frames in 2D
+    space.
+    """
```

Comment:
Again, I think it's enought to say that this is based on the `valid_poses_dataset` fixture, with the only difference being the length (5 frames instead of 10).